packages/esm-appointments-app/src/appointments/scheduled/appointments-list.component.tsx

```diff
@@ -2,14 +2,15 @@ import React from 'react';
 import { useAppointmentList } from '../../hooks/useAppointmentList';
 import AppointmentsTable from '../common-components/appointments-table.component';
 import { filterByServiceType } from '../utils';
+import dayjs from 'dayjs';
 
-interface AppointmentsByStatusProps {
+interface AppointmentsListProps {
   appointmentServiceType?: string;
-  status: string;
+  status?: string;
   title: string;
   date: string;
 }
-const AppointmentsByStatus: React.FC<AppointmentsByStatusProps> = ({ appointmentServiceType, status, title, date }) => {
+const AppointmentsList: React.FC<AppointmentsListProps> = ({ appointmentServiceType, status, title, date }) => {
```

Comment:
Not sure if "AppointmentsList" is the best name, but I renaming from "AppointmentsByStatus" because this will now fetch *all* appointments on a given day if you don't pass in a status, so the existing name seemed confusing.